src/library/librarycontrol.cpp

```diff
@@ -106,6 +106,12 @@ LibraryControl::LibraryControl(Library* pLibrary)
     connect(m_pAutoDjAddBottom.get(), SIGNAL(valueChanged(double)),
             this, SLOT(slotAutoDjAddBottom(double)));
 
+    // Sort controls
+    m_pSortColumn = std::make_unique<ControlPushButton>(ConfigKey("[Library]", "sort_column"));
+    m_pSortColumn->setButtonMode(ControlPushButton::TOGGLE);
```

Comment:
A toggle button changes between 1 and 0 each push, this matters for direct midi connections. I think this is not what we need here. ControlEncoder() is more useful. It passes the midi values through. 

Please set bIgnoreNops to false:
ControlEncoder(ConfigKey("[Library]", "sort_column"), false)
to allow to receive updates even the value is not changed. This can be used to toggle the sort order.

Reply:
Done, but I added a separate `sort_column_toggle` CO ([See below](https://github.com/mixxxdj/mixxx/pull/2118#discussion_r291200002)).